pillar.debian.example

```diff
@@ -2,6 +2,32 @@
 # vim: ft=yaml
 ---
 syslog_ng:
+  # tofs:
+  #   The files_switch key serves as a selector for alternative
+  #   directories under the formula files directory. See TOFS pattern
+  #   doc for more info.
+  #   Note: Any value not evaluated by `config.get` will be used literally.
+  #   This can be used to set custom paths, as many levels deep as required.
+  #   files_switch:
+  #     - any/path/can/be/used/here
+  #     - id
+  #     - roles
+  #     - osfinger
+  #     - os
+  #     - os_family
+  #   All aspects of path/file resolution are customisable using the options below.
+  #   This is unnecessary in most cases; there are sensible defaults.
+  #   path_prefix: template_alt
+  #   dirs:
+  #     files: files_alt
+  #     default: default_alt
+  #   The entries under `source_files` are prepended to the default source files
+  #   given for the state
+  #   source_files:
+  #     syslog_ng/config/install:
+  #       - 'example_alt.tmpl'
```

Comment:
Just the one entry here (i.e. remove the line underneath).  We tend to use a modified version of the actual template name, so something like:

```suggestion
  #       - 'alt_syslog-ng.conf'
```